### ISSUE TITLE
Individual audio levels

### ISSUE THREAD

**Issue by ottomated** [ottomated/CrewLink#6]
User should be able to right click on an avatar and adjust their output volume locally.

**Comment by ottomated** [comment]
In progress in #23

**Comment by edqx** [comment]
as if it's 2020 and you still don't have this

**Comment by Arxareon** [comment]
> as if it's 2020 and you still don't have this

yes. It's only been in progress since Nov 26 2020. Just be patient it will come.
